google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/GetAccountHierarchy.java

```diff
@@ -21,7 +21,7 @@
 import com.google.ads.googleads.v4.errors.GoogleAdsError;
```

Comment:
Can we change the commit message to have a bit more context about this change? I'd like to make sure that if anyone comes looking for the details, they can find our rationale for changing method signatures.